eventdisplay/tracks/FairEveTrack.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * FairEveTrack.h
+ *
+ *  Created on: 23 kwi 2020
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef FAIREVETRACK_H_
+#define FAIREVETRACK_H_
+
+#include <TEveTrack.h>
+#include <TEveTrackPropagator.h>
+#include <TVector3.h>
+
+class FairEveTrack : public TEveTrack
+{
+    Int_t fNPoints;
+    static Float_t fgWorldSize;
```

Comment:
I'd rather avoid new global state, can we find a better place to store the world size? Would a specialized `FairEveTrackPropagator : TEveTrackPropagator` be a suitable place to hold the world size?

Reply:
I'm not sure if this is solve issue. We still have to hold one global variable - FairEveTrackPropagator. Otherwise settings might be different for different tasks.

Reply:
> have to hold one global variable - FairEveTrackPropagator.

We should definitely avoid adding any C++ globals. Even global model parameters should be stored in C++ locals (e.g. as it is done with `TEveTrackPropagator`).

side notes:
* I realize FairRoot and ROOT are still full with unnecessary C++ globals, but if we never start avoiding them - at least with new code written in 2020 - then we will never get away from the problems they cause.
* If you wonder what's the problem with C++ globals, in a nutshell: they deny multithreading, limit composability with other libraries, and they may contribute to degraded performance scalability.
* As a rule of thumb, if possible, use C++ globals only in very rare cases, where you need to represent real global (computing) system resources. Physics model parameters, OTOH, are almost always avoidable as C++ globals.

---

Adding @MohammadAlTurany and @TobiasStockmanns to this discussion as they might have much deeper insight than me here on how to implement such a world size filter.

Reply:
Also refer to [I.2: Avoid non-const global variables](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#i2-avoid-non-const-global-variables)

Reply:
Actually I had another Idea - what about storing this variable in FairEventManager?
Theoretically this store information about geometry - therefore we could load this value from the size of the simulation - however I'm not sure how to get the size of the simulation. It seems that:
 Double_t Z = ((TGeoBBox*)gGeoManager->GetTopNode()->GetVolume()->GetShape())->GetDZ();
allows to get information about size directly from geoManager, however I'm not sure if TGeoBBox based class always represents the cave.